todo-app/src/App.js

```diff
@@ -1,13 +1,40 @@
-import React, { Component } from 'react';
+// import React, { Component } from 'react';
 
-class App extends Component {
+// class App extends Component {
+//   render() {
+//     return (
+//       <div>
+//         Hello World!
+//       </div>
+//     );
+//   }
+// }
+
+// export default App;
+
+import React from 'react';
+import Tasks from './Task'
+import task from './taskDB'
```

Comment:
This could be named plural `tasks` to avoid confusion. Also it actually represents and array of tasks.